examples/config/base/go-kprobe-counter/deployment.yaml

```diff
@@ -27,9 +27,11 @@ spec:
         name: go-kprobe-counter
     spec:
       nodeSelector: {}
-      hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: bpfman-app-go-kprobe-counter
+      securityContext:
+        runAsNonRoot: true
+        fsGroup: 65534
```

Comment:
Do we need anything in our documentation around this? If the user creates their own deployment, is 65534 a magic number, can they use any value they want?

Reply:
I was hoping you'd see that..... 65534 isn't magic but it is random :) 

I don't think we need to document anything as it's covered in upstream k8s docs https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
and should be pretty common for any user wishing to run an unprivileged application.